apps/dashboard/src/app/nebula-app/(app)/components/AssetsSection/AssetsSection.tsx

```diff
@@ -158,16 +173,85 @@ export function AssetsSection(props: {
         data: AssetBalance[];
       };
 
-      return json.data;
+      const tokensToShowOnTop = new Set(
```

Comment:
in future will we have some kind of API for this? feels like something that shouldn't forever be hard coded in frontend